src/api/script/import_database.rb

```diff
@@ -0,0 +1,126 @@
+# This script downloads database dump from a remote server and imports it into your local database.
+# It is a script and not a rake task, because downloading from a remote server usually needs
+# a private ssh key. As we're using vagrant in development this would mean that we need to
+# copy the private key into the vagrant box which we avoid with this script.
+require 'yaml'
+require 'fileutils'
+require 'optparse'
+require 'pathname'
+
+TABLES_TO_REMOVE = ['cache_lines', 'project_log_entries']
+params = {}
+
+OptionParser.new do |opts|
+  opts.banner = 'Usage: import_database.rb [options]'
+
+  opts.on('-a', '--all', 'Download and import the latest dump into development database.') do |v|
+    params[:all] = v
+  end
+
+  opts.on('-i', '--import', 'Import the latest dump into development database. You might want to specify the path with --filename.') do |v|
+    params[:import] = v
+  end
+
+  opts.on('-l', '--load', 'Download the latest dump into /db/data directory.') do |v|
+    params[:load] = v
+  end
+
+  opts.on('-p', '--path [PATH]', 'Specify the filename of the database dump. Default is /db/data/obs_production.sql.') do |v|
+    params[:path] = v
+  end
+
+  opts.on('-e', '--environment [PATH]', 'Specify the rails environment. Default is development.') do |v|
+    params[:environment] = v
+  end
+end.parse!
+
+
+def init(params)
+  unless Dir.pwd.include?('open-build-service')
+    abort('Please execute this script in your open-build-service directory')
+  end
+
+# Change in the open-build-service directory
+  while !Dir.pwd.end_with?('open-build-service')
```

Comment:
As discussed with @ChrisBr, consider this a nitpick. I would prefer if the script doesnt rely on the dir being named 'open-build-service' but since this is an internal tool we can always modify the script ourselves if we need to..